libs/core/src/lib/text/text.component.html

```diff
@@ -0,0 +1,17 @@
+<p class="fd-text" [class.fd-text--pre-wrap]="whitespaces" [style.hyphens]="hyphenation">
+    <span fd-lineclamp
+          class="fd-text__lineclamp"
+          #lineClamp="fdLineClamp"
+          [fdLineclampState]="_isCollapsed"
+          [fdLineClampLines]="maxLines!"
```

Comment:
why the `!` in `maxLines!`?

Reply:
This is an artefact from old implementation, I'll remove it